src/ngx_lua_resty_lmdb_module.h

```diff
@@ -9,6 +9,7 @@
 
 struct ngx_lua_resty_lmdb_conf_s {
     ngx_path_t  *env_path;
+    ngx_str_t   key_file;
```

Comment:
the variable and the `lmdb_encryption_key_file` directive needs renaming or rework of logic, we should either use it as a path (named `key_file`) and read its content, or use it as a inline key (so name it like `key_data` something), but not mix the two